docs/docs/install/docker-compose/index.md

```diff
@@ -249,7 +254,7 @@ Manually upgrade Mathesar to the newest version without using watch tower:
 
 ### Start Mathesar without the database server {#disable-db-service}
 
-The default `docker-compose.yml` has a `db` service which automatically starts a Postgres database server container called `mathesar_db`. This service allows you to immediately start using Mathesar to store data in a Postgres database without administering a separate Postgres server outside of Mathesar. But if you only plan to use Mathesar to connect to other Postgres servers, then you may disable Mathesar's inbuilt Postgres service if you like.
+The default `docker-compose.yml` has a `db` service which automatically starts a Postgres database server container called `mathesar_db`. This service allows you to immediately start using Mathesar to store data in a Postgres database without administering a separate Postgres server outside of Mathesar. But if you plan on using a different Database server, then you may disable Mathesar's inbuilt Postgres service if you like.
```

Comment:
Mathesar needs a database for storing its own data. "plan to use Mathesar to connect to other Postgres servers" was not highlighting that fact and was concerned about the user databases only.

Reply:
Cool, looks good.